modules/final_radiological_review/ajax/log.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Return History Log Json data
+ *
+ * PHP version 5.6
+ *
+ * @category Final_Ridiological_Review
+ * @package  Main
+ * @author   Wangshen <wangshen.mcin@gmail.com>
+ * @license  GPLv3 <http://www.gnu.org/licenses/gpl-3.0.en.html>
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ */
+ header("content-type:application/json");
+ ini_set('default_charset', 'utf-8');
+ $DB =& Database::singleton();
+
+   $headers = ['Time','User','Field','Old Value','New Value'];
+   
+            $history = $DB->pselect(
+                'SELECT  changeDate, userID, col, old, new 
+                 FROM final_radiological_review_history
+                WHERE CommentID= :primKey ORDER BY changeDate DESC',
+                array('primKey' => '300022MTL02222211468524343')
```

Comment:
I'm fairly sure this isn't supposed to be a hardcoded CommentID...

(This file also looks like it hasn't been run through PHPCS? The indentation is all off.)